src/web/src/app/app.module.ts

```diff
@@ -40,6 +39,12 @@ import {ConfirmationDialogComponent} from './confirmation-dialog/confirmation-di
 import {Title} from '@angular/platform-browser';
 import { UserService } from './user/service';
 import { SpaceService } from './space/service';
+import { SpaceAddComponent } from './space/add/component';
+import { SpaceDetailComponent } from './space/detail/component';
+import { SpaceListComponent } from './space/list/component';
+import { SpaceRequestSentComponent } from './space/request-sent/component';
+import { ProgressSpinnerComponent } from './progress-spinner/progress-spinner.component';
```

Comment:
Перенеси пожалуйста папку с progress-spinner
```suggestion
import { ProgressSpinnerComponent } from './request-watcher/progress-spinner/component';
```

Reply:
Готово.